test/run.sh

```diff
@@ -1,20 +1,20 @@
 #!/bin/bash
 
-set -e
+set -euo pipefail
```

Comment:
`-o pipefail` fails commands if an intermediate step fails.
`-u` is equivalent to `-o nounset`, which fails a script if it references unbound variables.